README.md

```diff
@@ -47,7 +47,7 @@ Make sure you have `virtualenv` installed.
 
 `--ignore_checks`: Ignore production checks
 
-`--unknown_arguments`: Pass any unknown arguments to the underlying command
+`--override_arguments`: Pass a list of arguments to the underlying command, overriding any set via the config file, specified as a space-separated list of \<key>=\<value> pairs
```

Comment:
Can you group these by cli parser?
`ignore_checks`, `unknown_arguments` and `bypass_prompt` were apart of the same parser.

Reply:
Can you add a section for this CliParser in the readme and give an example of usage?